legal-api/tests/unit/services/warnings/business/business_checks/test_firms.py

```diff
@@ -269,7 +271,11 @@ def test_check_firm_party(session, test_name, legal_type, role, party_type, expe
         party_role.related_colin_entity.organization_name = None
 
     with patch.object(firms, "check_address", return_value=[]):
-        result = check_firm_party(legal_type, party_role)
+        # TODO: check/update following block to work properly with AlternateName
+        if legal_type == AlternateName.EntityTypes.SP:
+            result = check_sp_party(legal_entity)
+        else:
+            result = check_gp_party(role)
```

Comment:
This is just to fix some `flake8` linting issues, I don't think it actually works. It will need to be fixed in the `services` unit test ticket.